prime-router/cleanslate.sh

```diff
@@ -31,6 +32,8 @@ OPTIONS:
   --prune-volumes           Forces a docker volume prune -f after taking containers
                             down (disables --keep-vault, including when set via --keep-all)
   --instructions            Shows post-run instructions
+  --retake-ownership        Change ownership of directories that are potentially 
```

Comment:
I would call this `--take-ownership` not `retake` (and then fix down as well)

Reply:
Done